build/spas.ts

```diff
@@ -325,30 +325,49 @@ export async function buildSPAs(options) {
   }
 }
 
-async function fetchRecentContributions() {
+async function fetchGitHubPRs(repo = "mdn/content", count = 5) {
```

Comment:
We don't really need a default value for repo here:

```suggestion
async function fetchGitHubPRs(repo, count = 5) {
```